frontend/src/app/shared/pipes/unit-transformation/unit-transformation.pipe.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { UnitTransformationPipe } from './unit-transformation.pipe';
+import { Unit } from '../../types/enums/Unit';
+
+describe('UnitTransformationPipe', () => {
+  it('create an instance', () => {
+    const pipe = new UnitTransformationPipe();
+    expect(pipe).toBeTruthy();
+  });
+
+  it('Format Percent label', () => {
+    const pipe = new UnitTransformationPipe();
+    expect(pipe.transform(1, Unit.PERCENT)).toBe('1%');
+  });
+
+  it('Format FTE label', () => {
+    const pipe = new UnitTransformationPipe();
+    expect(pipe.transform(1, Unit.FTE)).toBe('1 ' + Unit.FTE);
+  });
+
+  it('Format CHF label', () => {
+    const pipe = new UnitTransformationPipe();
+    expect(pipe.transform(1, Unit.CHF)).toBe('1 ' + Unit.CHF);
+  });
+
+  it('Format EUR label', () => {
+    const pipe = new UnitTransformationPipe();
+    expect(pipe.transform(1, Unit.EUR)).toBe('1 ' + Unit.EUR);
+  });
+
+  it('Format Number label', () => {
+    const pipe = new UnitTransformationPipe();
+    expect(pipe.transform(1, Unit.NUMBER)).toBe('1');
+  });
+
+  it('Format non-default label', () => {
```

Comment:
can this really happen?

Reply:
You're right, this currently never happens. It would also not necessarily play nice with i18n, so I've removed it for now. If it is ever needed, we can do a proper implementation.